index.html

```diff
@@ -5,86 +5,98 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <meta name="description" content="">
-    <meta name="keywords" content="">
-    <meta name="author" content="">
+    <meta name="description" content="Documentation for Primer CSS">
+    <meta name="keywords" content="CSS">
+    <meta name="author" content="GitHub Design Systems team">
 
     <title>
       Home &middot; Primer
     </title>
 
-    <link rel="stylesheet" href="/docs.css">
+    <link href="https://unpkg.com/primer-core/build/build.css" rel="stylesheet">
+    <link href="https://unpkg.com/primer-markdown/build/build.css" rel="stylesheet">
```

Comment:
I'd say not to rely on unpkg.com and just check in these build sources into this branch